src/cdk/bootstrap.ts

```diff
@@ -212,40 +210,50 @@ export class MiraBootstrap {
         ...process.env
       }
     })
-    await new Promise(resolve => {
-      proc.on('exit', () => {
+    await new Promise((resolve, reject) => {
```

Comment:
Is it possible here to catch the rejection (if it occurs) and provide a meaningful message before exiting (presumably with the more detailed error message)?

i.e. "Deploy CICD Failed.  Dumping Error message"

and then the CLI try/catch block presumably will exit with a non-zero code and dump the error details.

Reply:
I've just added an error message to each failing point to return the place where the error actually happened. 

Collecting the errors and print the dump is a bit difficult here, it's child process that has already printed the errors before exit.

If we would like to have a better error management we should probably consider a specific task that goes through the whole process find a proper way to collect the errors.

Reply:
Sounds good to me.  LGTM